hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -353,8 +353,11 @@ public void notifyConfigurationChanged(long term, long index,
 
   @Override
   public void pause() {
-    getLifeCycle().transition(LifeCycle.State.PAUSING);
-    getLifeCycle().transition(LifeCycle.State.PAUSED);
+    final LifeCycle lc = getLifeCycle();
```

Comment:
Are these changes connected to bumping ratis version?

Reply:
Yes, the tests are failing otherwise.